pebblo/app/service/service.py

```diff
@@ -143,9 +143,19 @@ def _set_anonymize_snippets(self):
         it will be used; otherwise, the value will be taken from the config.
         """
         if not self.data.get("anonymize_snippets"):
-            self.anonymize_snippets = config_details.get("classifier", {}).get(
-                "anonymizeSnippets", False
-            )
+            if config_details.get("reports", {}).get("anonymizeSnippets") is not None:
+                self.anonymize_snippets = config_details.get("reports", {}).get(
+                    "anonymizeSnippets"
+                )
+            elif (
+                config_details.get("classifier", {}).get("anonymizeSnippets")
+                is not None
+            ):
+                self.anonymize_snippets = config_details.get("classifier", {}).get(
+                    "anonymizeSnippets"
+                )
+            else:
+                self.anonymize_snippets = False
```

Comment:
This logic is duplicate. Can we move this whole function to a common place?

Reply:
This code block is repeated in `/loader/doc/` API file for storage type file and DB, currently we are not combining their code.